### ISSUE TITLE
Make Enable Night Mode more visible for users

### ISSUE THREAD

**Issue by SergeyZhukovsky** [brave/brave-browser#38553]
### Description

Some users mention that it's not easy to find night mode for web pages. We should probably enable it by default when Dark theme is on and remove Experimental from it's name. Probably name it different like `Enables dark mode for web pages content`.
https://community.brave.com/t/features-related-to-web-search-in-android-browser/528233

### Steps to reproduce

1. Go to Settings->Appearance
2. Observe `Enable "Night Mode" (Experimental)`

### Actual result

Enables dark mode for web content

### Expected result

Enables dark mode for web content

### Reproduces how often

Easily reproduced

### Brave version

1.65.x

### Device

- Brand/model: any
- Android version: any

### Channel information

- [X] release (stable)
- [X] beta
- [X] nightly

### Reproducibility

- [ ] with Brave Shields disabled
- [ ] with Brave Rewards disabled
- [ ] in the latest version of Chrome

**Comment by SergeyZhukovsky** [comment]
@timchilds do we want (2) apply to the current page only?

**Comment by deeppandya** [comment]
@timchilds @SergeyZhukovsky  we already have many items in the main menu. can we make the option enabled by default for now ? and add an option for each page later once we have menu redesign, we can add it in the menu. WDYT ?

**Comment by SergeyZhukovsky** [comment]
Yeah I agree, I would just stop on renaming and enabling with dark theme

**Comment by SergeyZhukovsky** [comment]
@timchilds as I understood from the community post the user is confused why he would need to switch some Night Mode after they turned on Dark Theme. The would expect Dark Theme does everything dark. Yeah I was thinking that Dark Theme that uses default OS setting would turn on web pages into dark mode as well.

**Comment by ahydronous** [comment]
Has there been more discussion on this?

It would be nice to have it sync with system theme, but just having it not disappear from the `...` menu after toggling off Night Mode would be a huge improvement already.

More broadly spoken: this should be an integrated consistent setting across all versions of Brave, including desktop Brave. Dark Reader works fine, but it'd be nice to have a native function built in.

**Comment by Brave-Matt** [comment]
Wondering what the status of this issue is - saw another user making a similar request here:
https://www.reddit.com/r/brave_browser/comments/1khwo3l/please_can_you_add_a_toggle_in_the_menu_of_brave/

Wondering if it is possible to add the theme button or even just a Night mode "toggle" in the main menu (similar to iOS) for easy access? As it is now, the setting itself - while not particularly hard to find - does require several taps and digging into the main settings menu to enable or disable the feature. 
@timchilds @SergeyZhukovsky
